tests/forte/forte_backbone_test.py

```diff
@@ -55,3 +90,59 @@ def test_import_forte_modules(self):
         from forte.process_job import ProcessJob, ProcessJobStatus
         from forte.process_manager import ProcessManager
         from forte.train_pipeline import TrainPipeline
+
+    def test_import_base_data_aug(self):
```

Comment:
will data aug require some additional dependencies?

Reply:
Only few modules under `forte/processors/data_augment` require the extra dependencies. For example,
`forte/processors/data_augment/algorithms/dictionary.py` requires `nltk`
`forte/processors/data_augment/algorithms/machine_translator.py` requires `transformers`

Reply:
I was also thinking maybe split modules under `forte/processors/data_augment` or other folders to two subfolders one requires extra dependencies and the other doesn't. I am not sure if it's good idea tho.

Reply:
But it seems that most of the data-aug algorithms in the `forte/processors/data_augment` folder are not in here: https://github.com/asyml/forte/blob/master/forte/processors/data_augment/algorithms/__init__.py. So I guess they won't be automatically imported anyway right?

Reply:
That's true. If we only need test from this perspective then `def test_import_base_data_aug(self):` is not needed.